build.sbt

```diff
@@ -222,26 +222,32 @@ lazy val `integration-test-local` = project
 
 lazy val docs = project
   .in(file("docs"))
-  .enablePlugins(AkkaParadoxPlugin)
+  .enablePlugins(AkkaParadoxPlugin, ParadoxSitePlugin, PublishRsyncPlugin)
   .disablePlugins(BintrayPlugin)
   .settings(
     name := "Akka Management",
     publish / skip := true,
     whitesourceIgnore := true,
+    makeSite := makeSite.dependsOn(LocalRootProject / ScalaUnidoc / doc).value,
+    previewPath := (Paradox / siteSubdirName).value,
     paradoxGroups := Map("Language" -> Seq("Java", "Scala")),
-    Compile / paradox := (Compile / paradox).dependsOn(LocalRootProject / ScalaUnidoc / doc).value,
+    Paradox / siteSubdirName := s"docs/akka-management/${if (isSnapshot.value) "snapshot" else version.value}",
     Compile / paradoxProperties ++= Map(
       "version" -> version.value,
-      "project.url" -> "https://developer.lightbend.com/docs/akka-management/current/",
-      "canonical.base_url" -> "https://developer.lightbend.com/docs/akka-management/current/",
+      "project.url" -> "https://doc.akka.io/docs/akka-management/current/",
+      "canonical.base_url" -> "https://doc.akka.io/docs/akka-management/current",
       "scala.binary_version" -> scalaBinaryVersion.value,
       "akka.version" -> Dependencies.AkkaVersion,
-      "extref.akka-docs.base_url" -> s"https://doc.akka.io/docs/akka/${Dependencies.AkkaVersion}/%s",
-      "extref.akka-http-docs.base_url" -> s"https://doc.akka.io/docs/akka-http/${Dependencies.AkkaHttpVersion}/%s.html",
+      "extref.akka-docs.base_url" -> s"https://doc.akka.io/docs/akka/current/%s",
+      "extref.akka-http-docs.base_url" -> s"https://doc.akka.io/docs/akka-http/current/%s",
+      "extref.akka-grpc-docs.base_url" -> s"https://doc.akka.io/docs/akka-grpc/current/%s",
```

Comment:
Is it okay to replace `Dependencies.AkkaVersion` with "current"? When Akka 2.6 is going to be out, older akka-management docs that were build against Akka 2.5 will start pointing to Akka 2.6 when that is current.

Reply:
The docs pointed to current [before](https://github.com/akka/akka-management/pull/553/files#diff-43949473e7d78d86e95a90eb610cc98aL158).
This was mostly to centralise that decision.
Pointing to the version will most of the time point to an earlier release. We could point to the minor release version.

Reply:
Pointing to a bit older version is fine. Then user gets the non-current version warning.